libretroshare/src/ft/ftserver.cc

```diff
@@ -1724,9 +1727,9 @@ int ftServer::handleIncoming()
 	// now File Input.
 	int nhandled = 0 ;
 
-	RsItem *item = NULL ;
+	RsItem *item = nullptr ;
 
-	while(NULL != (item = recvItem()))
+	while( (item = recvItem()) )
```

Comment:
Well, that is also reported by my compiler as a possible mistake, and I fully agree with him. Please keep the NULL (or change it to nullptr if you need)

Reply:
OK